content-api-crud/scripts/writeBulkEditBody.js

```diff
@@ -0,0 +1,29 @@
+// JSR223 script used internally by search-and-bulk-edit.yml
+// Reads list of IDs, Revision Ids and writes the request body to be used by
+// content-api bulk edit.
+
+var idList = new java.util.ArrayList(
```

Comment:
This is some interop from javascript to Java? For my own understanding, does the tool use Node? (does it have access to Node's 'fs', for example).

Reply:
It does not use Node. 
 https://stackoverflow.com/questions/45668451/jmeter-jsr-223-language-js-vs-javascript